client/components/SearchResult.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+export default class SearchResult extends React.Component {
+
+  static get propTypes () {
+    return {
+      myData: React.PropTypes.object

```

Comment:
I think you can take this out of the class as an object, along with the `export default`, so that we look everytime at the end of the file what it is exporting.

```
// class ends here
}

const { object } = React.PropTypes
SearchResult.propTypes = {
  myData: object
}

export default SearchResult
```